101.symmetric-tree.py

```diff
@@ -0,0 +1,38 @@
+#
+# @lc app=leetcode id=101 lang=python3
+#
+# [101] Symmetric Tree
+#
+
+# @lc code=start
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    def isSymmetric(self, root: TreeNode) -> bool:
+        """
+        time: O(N)  N : the number of all nodes
+        space: O(N)
+        """
+        if not root:
+            return True
+        else:
+            return self.isMirror(root.left, root.right)
+    def isMirror(self, left, right):
```

Comment:
insert a line before function definition.

Reply:
Check function name's character in PEP8. Unfortunately, function names in Leetcode break it, so you cannot use the same name style without checking coding style guides.